src/tests/SimTest.cpp

```diff
@@ -124,8 +164,116 @@ void SimTest::getPinholeCameraRGBAObservation() {
           Mn::PixelFormat::RGBA8Unorm,
           {pinholeCameraSpec->resolution[0], pinholeCameraSpec->resolution[1]},
           observation.buffer->data}),
-      Cr::Utility::Directory::join(TEST_DIR, "SimTestExpected.png"),
-      (Mn::DebugTools::CompareImageToFile{15.0f, 0.17f}));
+      Cr::Utility::Directory::join(screenshotDir, groundTruthImageFile),
+      (Mn::DebugTools::CompareImageToFile{30.f, 0.5f}));
+}
+
+void SimTest::getSceneRGBAObservation() {
+  setTestCaseName(CORRADE_FUNCTION);
+  auto simulator = getSimulator(vangogh);
+  checkPinholeCameraRGBAObservation(*simulator, "SimTestExpectedScene.png");
+}
+
+void SimTest::getDefaultLightingRGBAObservation() {
+  setTestCaseName(CORRADE_FUNCTION);
```

Comment:
`setTestCaseName()` is strictly necessary only in the `getSceneRGBAobservation()` above, in the others the `CORRADE_VERIFY()` macros are early enough to take care of extracting the correct function name.

I tried to find a way around having to explicitly call this, but it looks like the approach you went with here was the best possible :) The test cases are not similar enough to make use of [instanced tests](https://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#TestSuite-Tester-instanced) and OTOH putting the common sensor & agent setup into a [setup / teardown function](https://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#TestSuite-Tester-setup-teardown) would unnecessarily complicate things.